Presentation.Web/Controllers/API/V2/External/ItInterfaces/ItInterfaceV2Controller.cs

```diff
@@ -307,6 +307,8 @@ public IHttpActionResult DeactivateItInterfaceAsRightsHolder([NonEmptyGuid] Guid
         /// <param name="changedSinceGtEq">Include only changes which were LastModified (UTC) is equal to or greater than the provided value</param>
         /// <param name="nameEquals">Include only interfaces with a name equal to the parameter</param>
         /// <param name="usedInOrganizationUuid">Filter by UUID of an organization which has taken the related it-system into use through an it-system-usage resource</param>
+        /// <param name="interfaceId">Include only interfaces with an ItInterfaceId equal to the parameter</param>
```

Comment:
remember that on the v2 it's not colled itinterfaceid but InterfaceId.. so change the text here